packages/clou-utils/src/generalUtils.js

```diff
@@ -35,22 +38,24 @@ function isFalse(valueToCheck) {
   return !isTrue(valueToCheck);
 }
 
-function encrypt(data) {
+async function encrypt(data) {
+  const encryptionKey = await getParameterFromSSM(encryptionKeyPath);
   const iv = crypto.randomBytes(IV_LENGTH);
-  const cipher = crypto.createCipheriv(encryptionAlgorithm, Buffer.from(encryptionKey), iv);
+  const cipher = crypto.createCipheriv(encryptionAlgorithm, Buffer.from(JSON.parse(encryptionKey)), iv);
   let encryptedData = cipher.update(data, inputEncoding, outputEncoding);
   encryptedData += cipher.final(outputEncoding);
   const authTag = cipher.getAuthTag();
   return `${iv.toString(outputEncoding)}${encryptedData}${authTag.toString(outputEncoding)}`;
 }
 
-function decrypt(data) {
+async function decrypt(data) {
+  const encryptionKey = await getParameterFromSSM(encryptionKeyPath);
```

Comment:
why not remove the default encryption-key and force the "clients" of this code to have the key on the process.env (i.e. reading it using custom-environment-variables)?